documentation/dashboard/working_with_data.md

```diff
@@ -439,6 +439,174 @@ Create New Report
+2020-01-01 13:00:00+00:00,140,""
+2020-01-01 13:15:00+00:00,150,""
+2020-01-01 13:30:00+00:00,160,""
+2020-01-01 13:45:00+00:00,170,""
+```
+
+
+#### Discarding intervals with incomplete data
+
+We can use quality flags to exclude intervals that contain too much undesirable data. For
+example, if we are resampling to one hour intervals and don't want to include
+hours where half or more of the data occurs at night time, we can use a filter
+like:
+
+```
+Quality Flags: "NIGHTTIME"
+Discard Before Resample: False
+Resample Threshold Percentage: 50%
+```
+
+the observation data above is filtered and resampled into:
+```
+timestamp,value
+2020-01-01 13:00:00+00:00,155.0
+```
+The first and second hour interval is dropped because 100% and 50% of the datapoints
+in those intervals are flagged with `NIGHTTIME`, *meeting* or *exceeding* the
+resample threshold.
+
+If we want to include intervals where half of the data are
+flagged with `NIGHTTIME`, we could adjust the resample threshold
+percentage to 51%. The threshold must be *met or exceeded* for an interval
+to be excluded.
+
+Using the filter:
+```
+Quality Flags: "NIGHTTIME"
+Discard Before Resample: False
+Resample Threshold Percentage: 51%
+```
+the observation data above is filtered and resampled into:
+```
+timestamp,value
+2020-01-01 12:00:00+00:00,235.0
+2020-01-01 13:00:00+00:00,155.0
+```
+
+#### Discarding erroneous data
+
+If we want to remove erroneous data and exclude it from the resampling
+process, we can set discard before resample to True. 
+
+Using the filter:
+```
+Quality Flags: "USER FLAGGED"
+Discard Before Resample: True
+Resample Threshold Percentage: 50%
+```
+
+the observation data above is filtered and resampled into: 
+```
+timestamp,value
+2020-01-01 12:00:00+00:00,113.33
+2020-01-01 13:00:00+00:00,155.0
+```
+
+In this case, the first interval is included in the resampled data because only
+one point is flagged with `USER FLAGGED`, which does not exceed our 50% resample
+threshold. However, the datapoint flagged with `USER FLAGGED` was dropped before
+resampling occured which reduces the interval average from 235.0 to 113.33.
+
+#### Filtering for multiple flags in a single filter
+
+When listing multiple flags in a single filter, the number of unique datapoints
+flagged with *any* of the listed quality flags determines if the
+resample threshold percentage is exceeded or not.
+
+Using the filter: 
+```
+Quality Flags: "NIGHTTIME, USER FLAGGED, "
+Discard Before Resample: False
+Resample Threshold Percentage: 75%
+```
+
+the observation data above is filtered and resampled into:
+```
+timestamp,value
+2020-01-01 13:00:00+00:00,155.0
+```
+The first hour interval is dropped because 100% of timestamps in
+the first hour are flagged `NIGHTTIME`. The second hour interval
+is excluded because 75% of the points are  flagged with either
+`NIGHTTIME`, or `USER FLAGGED`, which meets the resample threshold.
+
+#### Excluding intervals where all data is flagged
```

Comment:
Maybe going in circles, but should this be something more like "Mixing filter types"?